packages/ui-causal-graph-editor/src/shared/use-pane-visibility.tsx

```diff
@@ -15,6 +15,10 @@ export default function usePaneVisibility(
     const isPaneVisible = React.useRef(false);
 
     React.useEffect(() => {
+        if (!pane.current) {
```

Comment:
Should this effect have `pane.current` as a dep?

Reply:
Ref values shouldn't be deps, they're not reactive. It would have no effect